hrjobcontract/js/src/job-contract/directives/loader.js

```diff
@@ -1,56 +1,57 @@
+/* eslint-env amd */
+
 define([
-    'job-contract/directives/directives'
+  'job-contract/directives/directives'
 ], function (directives) {
-    'use strict';
-
-    directives.directive('hrjcLoader',['$rootScope','$log', function ($rootScope, $log) {
-        $log.debug('Directive: hrjcLoader');
+  'use strict';
 
-        return {
-            link: function ($scope, el, attrs) {
-                var loader = document.createElement('div'),
-                    loaderSet = false,
-                    positionSet = false;
+  directives.directive('hrjcLoader', ['$rootScope', '$log', function ($rootScope, $log) {
+    $log.debug('Directive: hrjcLoader');
 
-                loader.className = 'hrjc-loader';
+    return {
+      link: function ($scope, el, attrs) {
+        var loader = document.createElement('div');
+        var loaderSet = false;
+        var positionSet = false;
 
-                function isPositioned(){
-                    var elPosition = window.getComputedStyle(el[0]).position;
-                    return elPosition == 'relative' || elPosition == 'absolute' || elPosition == 'fixed'
-                }
+        loader.className = 'hrjc-loader spinner';
 
-                function appendLoader() {
-                    if (!isPositioned()) {
-                        el.css('position','relative');
-                        positionSet = true;
-                    }
+        function isPositioned () {
+          var elPosition = window.getComputedStyle(el[0]).position;
```

Comment:
`elPosition` sounds like the code is written in another language. I think it is better to use `elementPosition`. Same applies to everywhere else `el` is used.